### PR TITLE
fix(mobile): prepare mobile app for release

### PR DESCRIPTION
**Pull request by VladislavSokov**

### CODE REVIEW

File: mobile/App.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ import Config from './config';
 export default function App() {
   return (
     <WebView
-      style={{ marginTop: 25 }}
```
Comment (ka8725) [inline mobile/App.js:7]: why changed?

**Comment by ka8725** [comment]
@VladislavSokov can we also change the icon? I think it has wrong size. It's too big for the mobile.

Another question - is it possible to change the loading image instead of expo icon show our icon.

File: mobile/App.js (hunk 1 above)
Comment (VladislavSokov) [reply mobile/App.js:7]: not needed

**Comment by VladislavSokov** [comment]
> @VladislavSokov can we also change the icon? I think it has wrong size. It's too big for the mobile.
> 
> Another question - is it possible to change the loading image instead of expo icon show our icon.

I'll try my best

**Pull request merged**